javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,11 @@
+/* exported filterOutNulls */
+function filterOutNulls(values) {
+  const noNullsArray = [];
+
+  for (let i = 0; i < values.length; i++) {
+    if (values[i] != null) {
+      noNullsArray.push(values[i]);
+    } else;
```

Comment:
If you don't plan on using the `else` statement, there's no reason to include it! It's not required

Reply:
Ah okay, thanks for the tip!